src/components/HeroSection.vue

```diff
@@ -74,6 +74,7 @@ export default {
   methods: {
     onSubmit() {
       this.$store.commit(SET_QUERY, { query: { q: this.form.searchTerm }, shouldNavigate: true });
+      this.$store.commit(SET_IMAGES, { images: [] });
```

Comment:
I think it's probably worth trying to set the `images: []` directly in the `SET_QUERY` mutation handler. You can try changing this and I'll make tests here to see if it doesn't cause any issues

Reply:
Sure, I'll give it a try